src/edit_distance.jl

```diff
@@ -82,21 +82,20 @@ function edit_distance(G₁::SimpleGraph, G₂::SimpleGraph;
 end
 
 function is_complete_path(λ, G₁, G₂)
-  us = []; vs = []
+  us = Set(); vs = Set()
   for (u,v) in λ
     push!(us, u)
     push!(vs, v)
   end
+  delete!(us, 0)
+  delete!(vs, 0)
 
-  us = unique(us[us .> 0])
-  vs = unique(vs[vs .> 0])
-
-  length(us) == nv(G₁) && length(vs) == nv(G₂)
+  return length(us) == nv(G₁) && length(vs) == nv(G₂)
 end
 
 function DefaultEditHeuristic(λ, G₁::SimpleGraph, G₂::SimpleGraph)
-  vs = [v for (u,v) in λ]
-  vs = unique(vs[vs .> 0])
+  vs = Set([v for (u,v) in λ])
```

Comment:
I think this is ok, but it might be allocating a lot of memory if `λ` is large. Using a `push!(vs...)` inside a `for` loop might be more effective, as it won't create/collect a vector first.